xbmc/input/ButtonTranslator.cpp

```diff
@@ -1195,6 +1195,38 @@ CAction CButtonTranslator::GetGlobalAction(const CKey &key)
   return GetAction(-1, key, true);
 }
 
+bool CButtonTranslator::HasLonpressMapping(int window, const CKey &key)
+{
+  map<int, buttonMap>::const_iterator it = m_translatorMap.find(window);
+  if (it == m_translatorMap.end())
+  {
+    if (window > -1)
+      return HasLonpressMapping(GetFallbackWindow(window), key);
```

Comment:
I think this results in a stack overflow for every dynamic addon window. GetFallbackWindow never returns -1 in this case